modules/product/product.service.js

```diff
@@ -210,22 +210,33 @@ class ProductsService {
       throw new RuleError(PRODUCT_HAS_NOT_CHANGED, FORBIDDEN);
     }
     if (primary) {
```

Comment:
please remove nested if statement, it is not necessary if now

Reply:
Fixed